react-multiplatform-app/src/pages/Hardware.tsx

```diff
@@ -0,0 +1,41 @@
+import {Button, StyleSheet, View} from "react-native";
+import {useNavigation} from "@react-navigation/native";
+import React from "react";
+
+const Hardware = () => {
+  const navigation = useNavigation();
+
+  const buttons = [
+    { id: 10, title: 'Accelerometer', routeName: 'Accelerometer' },
+    { id: 20, title: 'Camera', routeName: 'Camera' },
+    { id: 30, title: 'Gps', routeName: 'Gps' },
+    { id: 40, title: 'Light', routeName: 'Light' },
+    { id: 50, title: 'Microphone', routeName: 'Microphone' },
```

Comment:
I think in android I have made additional category here, named sensors and inside I put light and Accelerometer.

Reply:
Sensors are named "hardware" in my app, will change